### ISSUE TITLE
Flood lights not on when only bat on and stby pwr auto

### ISSUE THREAD

**Issue by MCCVeen** [mSparks43/747-400#164]
See references below
![747-400 no flood lights bat+stby pwr](https://user-images.githubusercontent.com/75425109/101568926-cde2af00-39d3-11eb-844f-a5098df91927.png)
![747-400 no flood lights bat+stby pwr 1](https://user-images.githubusercontent.com/75425109/101568980-e6eb6000-39d3-11eb-83b9-a97340aec811.png)
![image](https://user-images.githubusercontent.com/75425109/101568985-e94dba00-39d3-11eb-96a4-eb450f3bfc36.png)

**Comment by mSparks43** [comment]
Please retest, can't recreate

**Comment by MCCVeen** [comment]
![image](https://user-images.githubusercontent.com/75425109/105157541-5ee99180-5b0d-11eb-9851-ce811f4acd0a.png)
![image](https://user-images.githubusercontent.com/75425109/105157561-65780900-5b0d-11eb-96f4-09cd57fb541d.png)

**Comment by MCCVeen** [comment]
All flood lights should be on with standby power in auto

**Comment by crazytimtimtim** [comment]
Isn't that the X-Plane flashlight? I don't think the 744 has red lights...

X-Plane has two flashlights, a red one and a normal, more white, one.

**Comment by MCCVeen** [comment]
> > All flood lights should be on with standby power in auto without electrical power
> 
> The FCOM never stated they _will_ be on with standby power. It just said they will be _powered_ from standby power. So _if_ you turn them on with only standby power, they will work, but that doesn't necessarily mean they are _always_ on. Correct, @mSparks43?

No that's not correct. They'll turn on to give the pilots lightning in case they're flying and it's pitch black outside and they can't see anything

**Comment by crazytimtimtim** [comment]
Ok.

**Comment by mSparks43** [comment]
> without electrical power

What does this mean? "without electrical power" from where?

**Comment by MCCVeen** [comment]
> > without electrical power
> 
> What does this mean? "without electrical power" from where?

Any generators (engines, apu) and external power.

**Comment by mSparks43** [comment]
> Any generators (engines, apu) and external power.

Sounds like, if the elec page doesn't look _exactly_ like this this flood lights are on.
![image](https://user-images.githubusercontent.com/6773686/116868495-262e7d80-ac0f-11eb-8eeb-4880135a5fca.png)

**Comment by MCCVeen** [comment]
> > Any generators (engines, apu) and external power.
> 
> Sounds like, if the elec page doesn't look _exactly_ like this this flood lights are on.
> ![image](https://user-images.githubusercontent.com/6773686/116868495-262e7d80-ac0f-11eb-8eeb-4880135a5fca.png)

No, when there is no generator at all, so no power supply active, so say you only turned on the battery and the standby power, all the flood lights will be activated in case of emergency

**Comment by mSparks43** [comment]
> No, when there is no generator at all, so no power supply active, so say you only turned on the battery and the standby power, all the flood lights will be activated in case of emergency

but that doesn't describe when they turn off

**Comment by MCCVeen** [comment]
> > No, when there is no generator at all, so no power supply active, so say you only turned on the battery and the standby power, all the flood lights will be activated in case of emergency
> 
> but that doesn't describe when they turn off

Just set this issue to the side for now, I'll see if I can get a full explanation on exactly what lights turn on in which scenario somewhere this week since this is a pretty complicated thing

**Comment by crazytimtimtim** [comment]
Good question for @V1ROTA7E ?

**Comment by crazytimtimtim** [comment]
From what I can tell, the floodlights will come on as long as you have DC power (Battery) but no AC power (generator).

**Comment by MCCVeen** [comment]
> From what I can tell, the floodlights will come on as long as you have DC power (Battery) but no AC power (generator).

Exactly, now the difficult part is to find out exactly which lights turn on and at what setting. I tried it out in the PMDG and some panel lights come on and some don't, like for example the Captain EFIS lights will turn on but not the F/O's or the MCP so it's a bit confusing

**Comment by crazytimtimtim** [comment]
Sounds about right. This is a 737 video but it's still similar. Captain's instruments stay, FO's don't.

https://www.youtube.com/watch?v=JamDKcYUSu8
